types/wx-js-sdk-dt/index.d.ts

```diff
@@ -0,0 +1,468 @@
+// Type definitions for wx-js-sdk 1.2
+// Project: https://res.wx.qq.com/open/js/jweixin-1.2.0.js
+// Definitions by: Bian Zhongjie <https://github.com/agasbzj>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export namespace wx {
```

Comment:
Just a question: Is this only usable as a global in a browser, or should it also be usable as a module? If it is only usable as a global, then this construction is fine. If not, then you should remove the namespace surrounding all of these members, add `export` modifiers to them, then add `export as namespace wx` to the file to make it usable both as a global and as a module with `import`.